Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/RelationshipsStepDefinitions.cs

```diff
@@ -187,6 +219,23 @@ public async Task ThenTheResponseContainsARelationship()
         }
     }
 
+    [Then("a relationship can be established")]
+    public void ThenARelationshipCanBeEstablished() => PerformCanEstablishRelationshipCheck(true);
```

Comment:
Boolean parameters are really ugly. It's so hard to understand what they are used for.

I think I'd rather duplicate the code than having the boolean parameter.